ebuild-writing/common-mistakes/text.xml

```diff
@@ -404,8 +404,9 @@ the guidelines to use TABS rather than spaces. So <e>please</e> use tabs!
 <body>
 
 <p>
-I'm often guilty of this. Remember to run repoman over your ebuilds so it can
-tell you if there is trailing whitespace at the end of lines or on empty lines.
+I'm often guilty of this. Remember to run <c>repoman</c> or <c>pkgcheck</c>
+over your ebuilds so it can tell you if there is trailing whitespace at the end
+of lines or on empty lines.
```

Comment:
If you mention several tools, then don't forget app-emacs/ebuild-mode (and app-xemacs/ebuild-mode) which will automatically trim trailing whitespace. :smile:

Reply:
Sure, no problem!